charts/images.yaml

```diff
@@ -2,7 +2,7 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "v0.14.0"
+  tag: "v0.15.0-dev-c75d4a090e631fa10787f396bd676e6c6147eb41"
```

Comment:
Please remember to revert this before merging

Reply:
done